tests/pipeline/test_pipeline.py

```diff
@@ -34,6 +34,35 @@ def test_load_pipeline(self, snapshot: SnapshotTest):
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
+    def test_name_equal_prefix_name_concatenation(self):
+
+        raw_pipeline = yaml.safe_load(
+            open(str(RESOURCE_PATH / "name_prefix_concatenation/pipeline.yaml"), "r")
+        )
+
+        result = runner.invoke(
+            app,
+            [
+                "generate",
+                "--pipeline-base-dir",
+                PIPELINE_BASE_DIR,
+                str(RESOURCE_PATH / "name_prefix_concatenation/pipeline.yaml"),
+                "tests.pipeline.test_components",
+                "--defaults",
+                str(RESOURCE_PATH),
+            ],
+            catch_exceptions=False,
+        )
+
+        assert result.exit_code == 0
+
+        enriched_pipeline = yaml.safe_load(result.stdout)
+
+        assert (
+            enriched_pipeline["components"][0]["name"]
+            == raw_pipeline[0]["prefix"] + raw_pipeline[0]["name"]
```

Comment:
Make the assert explicit, i.e., put "my-fake-prefix-my-streams-app"

Reply:
you were faster than me :)